code/api/src/modules/user/mutations.js

```diff
@@ -2,8 +2,8 @@
 import { GraphQLString, GraphQLInt } from 'graphql'
 
 // App Imports
-import { UserType } from './types'
-import { create, remove } from './resolvers'
+import { UserType, UserType1 } from './types'
```

Comment:
This variable name is going to have to change in the future. I recommend `UserUpdateType`

Reply:
Great job with digging into the code examples to get the correct User Type.